www/js/suggestions/suggestions-factory.js

```diff
@@ -206,12 +206,29 @@ angular.module('starter.suggestions-factory', ['ionic', 'starter.filter-factory'
   suggestions.setAsGoing = function(suggestionId) {
     var suggestionsLength = suggestions.options.length;
     for (var i = 0; i < suggestionsLength; i++) {
-        if (suggestionId == suggestions.options[i].id) {
+        if (suggestionId == suggestions.options[i].id &&
```

Comment:
i don't think you need to add an extra check here. if we want to mark a suggestion as going then we should be able to set it as going even if we have been there already. the definition description of the function is to set a suggestion with supplied `id` as going.